### ISSUE TITLE
Loss NaN for Multi-GPU Training (GeForce GTX-1080 Ti)

### ISSUE THREAD

**Issue by SakhawatSumit** [SeanNaren/deepspeech.pytorch#315]
I was training with a custom dataset, it works fine with a single Ge-Force GTX 1080 Ti gpu. But after few mini-batch loss is Nan for multi-gpu. 
Pytorch version 0.4.0
Is there any solution?

**Comment by ryanleary** [comment]
Have you made any changes related to the learning rate? What is your batch size? How much data is there? There's a large array of things that could cause that to happen and you haven't really provided enough information to help troubleshoot.

**Comment by SakhawatSumit** [comment]
I used batch size 32 with all default hyper-parameters. Data is about 500K with maximum 10s long audio sample.

**Comment by hoyden** [comment]
Maybe your learning rate set too big
